modules/optimization/doc/content/modules/optimization/examples/forceInv_pointLoads.md

```diff
@@ -0,0 +1,124 @@
+
+!media media/fig_setup.png
+       style=width:40%;margin:auto;padding-top:2.5%;
+       id=figSetup
+       caption=Meshed geometry with measurement ($\times$) and parameterized point load ($\bigcirc$) locations.
+
+## Main Application Input
+
+Optimization problems are solved using the [MultiApps](MultiApps/index.md) system.  The main application contains the optimization executioner and the sub-applications solve the forward and adjoint PDE.   The main application input is shown in [fig:master_app].
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/main.i
+         id=fig:master_app
+         caption=Main application optimization input for point load parameterization shown in [figSetup]
+
+The main application runs the optimization executioner and transfers data from the optimization executioner back and forth to the sub-apps that are running the "forward" and "adjoint" FE models.  Since no mesh or physics kernels are required on the main-app, we use the [StochasticTools](syntax/StochasticTools/index.md) action to set up all of the nullkernels, empty mesh, etc. needed to get a MOOSE simulation to run.
+
+The [optimize](Optimize.md) executioner block, shown below, provides an interface with the [PETSc TAO](https://www.mcs.anl.gov/petsc/documentation/taosolvertable.html) optimization library. The optimization algorithm is selected with `tao_solver` with specific solver options set with `petsc_options`.
+
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/main.i
+         block=Executioner
+
+The `Optimize` executioner requires a [OptimizationReporter](syntax/OptimizationReporter/index.md), shown below, to transfer data between the optimization executioner and the transfers used for communicating with the sub-apps.  The type of `OptimizationReporter` depends on the optimization algorithm, given by teh
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/main.i
+         block=OptimizationReporter
+
+
+
+The optimization executioner on the main app does not need a mesh or variables and uses the action to set-up a dummy mesh and variables.
+
+!media media/fig_forward.png
+      style=width:60%;margin:auto;padding-top:2.5%;
+      id=figForward
+      caption=Optimization results for the (a) temperature field being matched at the $\times$ symbols in [figSetup] and (b) the intensity of the parameterized point loads applied at the $\bigcirc$ needed to produce the temperature field in (a)
+
+!media media/fig_adjoint.png
+      style=width:60%;margin:auto;padding-top:2.5%;
+      id=figAdjoint
+      caption=Adjoint temperature field and misfit load from first optimization iteration.
+
+The first step is to define the simulation to perform, in this case the simulation considered is a 2D
+steady state diffusion equation with Dirichlet boundary conditions on each end of the domain: find $u$
+such that
+
+!equation
```

Comment:
Confused here: the word above says a 2D steady state diffusion equation, but here given as a transient 1D parabolic equation. Copy-paste error?